packages/datadog-plugin-aerospike/test/naming.js

```diff
@@ -0,0 +1,19 @@
+const { resolveNaming } = require('../../dd-trace/test/plugins/helpers')
+
+const rawExpectedSchema = {
+  command: {
+    v0: {
+      opName: 'aerospike.command',
+      serviceName: 'test'
```

Comment:
As discussed, let's focus on consistent behaviour. The whole point of v1 is to fix the problem in a consistent manner, but in the meantime, v0 services should be consistent with each other.